src/css/doc.css

```diff
@@ -776,7 +776,8 @@
 .doc .collapseBox .content {
   display: none;
   font-weight: 300;
-  font-size: calc(16/var(--rem-base)*1rem);
+  font-size: 1.05em;
+  margin-top: 1rem;
```

Comment:
This doesn't match the machbar design. The content of the collapse boxes was fine I think, but the title font was too big and the margin between the collapse box and preceding paragraph too small.